week1/homework/test/routes.spec.js

```diff
@@ -0,0 +1,83 @@
+const request = require("supertest");
+const app = require("../src/backend/app");
+const meals = require("../src/backend/data/meals");
+const reviews = require("../src/backend/data/reviews");
+const reservations = require("../src/backend/data/reservations");
+
+describe("GET /", () => {
+    test("responds with corrected response", async () => {
+        const response = await request(app).get("/");
+        expect(response.statusCode).toBe(200);
+        expect(response.text).toBe("Meal Sharing Web App");
+    });
+});
+
+describe("GET /meals", () => {
+    test("responds with all meals (with their reviews, if any) from data file", async () => {
```

Comment:
Could patch the test data for all of these tests (to control return values and to improve uniformity), instead of using the jsons from the `data` directory, but this would likely be confusing when running the tests, as it would be unclear where the data would be coming from.